common/src/app/common/flags.cljc

```diff
@@ -7,13 +7,150 @@
+    :login-with-ldap
+    ;; disabled by default. Uses any generic authentication provider that implements OIDC protocol as credentials.
+    :login-with-oidc
+    ;; disabled by default.
+    :oidc-registration
+    ;; disabled by default. This logs to console the invitation tokens. It's useful in case the SMTP is not configured.
+    :log-invitation-tokens})
+
+(def email
+  "Flags related to email features"
+  #{
+    ;; disabled by default. Uses the domains in whitelist as the only allowed domains to register in the application.
+    ;; used with PENPOT_REGISTRATION_DOMAIN_WHITELIST
+    :email-whitelist
+    ;; disabled by default. Prevents the domains in blacklist to register in the application.
+    ;; used with PENPOT_REGISTRATION_DOMAIN_BLACKLIST
+    :email-blacklist
+    ;; enabled by default. Skips the email verification process. Not recommended for production environments.
+    :email-verification
+    ;; disabled by default ¿?. Only used if SMTP is disabled. Logs the emails into the console.
+    :log-emails
+    ;; disabled by default. Enable it to configure email settings.
+    :smtp
+    ;; ¿?
+    :smtp-debug})
+
+(def varia
+  "Rest of the flags"
+  #{
+    ;; ¿?
+    :audit-log
+    ;; ¿?
+    :audit-log-archive
+    ;; ¿?
+    :audit-log-gc
+    ;; ¿?
+    :auto-file-snapshot
+    ;; enables the `/api/doc` endpoint that lists all the rpc methods available.
+    :backend-api-doc
+    ;; ¿?
+    :backend-openapi-doc
+    ;; ¿?
+    :backend-worker
+    ;; ¿?
+    :component-thumbnails
+    ;; enables the default cors configuration that allows all domains (currently this configuration is only used for development).
+    :cors
+    ;; ¿?
+    :dashboard-templates-section
+    ;; disabled by default. When enabled, Penpot create demo users with a 7 days expiration.
+    :demo-users
+    ;; disabled by default. When enabled, it displays a warning that this is a test instance and data will be deleted periodically.
+    :demo-warning
+    ;; ¿?
+    :file-schema-validation
+    ;; ¿?
+    :soft-file-schema-validation
+    ;; ¿?
+    :file-validation
+    ;; ¿?
+    :soft-file-validation
+    ;; enabled by default. ¿?
+    :frontend-svgo
+    ;; enabled by default. ¿?
+    :exporter-svgo
+    ;; enabled by default. ¿?
+    :backend-svgo
+    ;; ¿?
+    :google-fonts-provider
+    ;; ¿?
+    :nrepl-server
+    ;; enabled by default. Shows the onboarding modals right after registration.
+    :onboarding
+    ;; ¿?
+    :quotes
+    ;; ¿?
+    :soft-quotes
+    ;; ¿?
+    :renderer-v2
+    ;; ¿? does this need a soft-rpc-climit?
+    :rpc-climit
+    ;; ¿?
+    :rpc-rlimit
+    ;; ¿?
+    :soft-rpc-rlimit
+    ;; enabled by default. Disable it if you want to serve Penpot under a different domain than `http://localhost` without HTTPS.
+    :secure-session-cookies
+    ;; ¿?
+    :strict-session-cookies
+    ;; it seems redundant with the PENPOT_ENABLE_TELEMETRY option ¿?
+    :telemetry
```

Comment:
Yes, this is the way to setup the telemetry with flags. But the `PENPOT_ENABLE_TELEMETRY` is preserved for backward compatibility. And we are not going to remove it so yes it is redundant.